index.bs

```diff
@@ -1,16 +1,165 @@
 <pre class='metadata'>
-Title: Your Spec Title
-Shortname: your-spec
+Title: Conversion Measurement
+Shortname: conversion-measurement
 Level: 1
-Status: ED
-Group: WGNAMEORWHATEVER
-URL: http://example.com/url-this-spec-will-live-at
-Editor: Your Name, Your Company http://example.com/your-company, your-email@example.com, http://example.com/your-personal-website
-Abstract: A short description of your spec, one or two sentences.
+Status: CG-DRAFT
+Group: wicg
+Repository: WICG/conversion-measurement-api
+URL: https://wicg.github.io/conversion-measurement-api
+Editor: Charlie Harrison, Google Inc. https://google.com, csharrison@chromium.org
+Abstract: A new API to measure and attribute cross-site conversions.
+
+Markup Shorthands: markdown on
+Complain About: accidental-2119 on, missing-example-ids on
+Assume Explicit For: on
 </pre>
 
 Introduction {#intro}
 =====================
 
-Introduction here.
+<em>This section is non-normative</em>
+
+This specification describes how web browsers can provide a mechanism to the
+web that allows measuring and attributing conversions (e.g. purchases) to ads
+a user interacted with on another site. This mechanism should remove one need
+for cross site identifiers like third party cookies.
+
+## Overview ## {#overview}
+
+An anchor tag with <{a/impressiondata}> and <{a/conversiondestination}> attributes is
+classified as an <dfn export>impression tag</dfn>. When impression tags are clicked, and the
+resulting navigation commits in a document matching the <{a/conversiondestination}>,
+then the impression is stored in UA storage.
+
+At a later point, the <{a/conversiondestination}> site may fire an HTTP request to
+trigger conversion registration, which matches up with any previously
+stored impressions. If matching impressions exist, they are scheduled to be
+reported at a later time, possibly multiple days in the future.
+
+Reports are sent to reporting endpoints that are configured in impression tags
+and conversion registration requests.
+
+# Fetch monkeypatches # {#fetch-monkeypatches}
+
+TODO: Patch into fetch for cancelling requests redirected to the .well-known
+conversion domain.
+
+# HTML monkeypatches # {#html-monkeypatches}
+
+Rewrite the anchor element to accept the following attributes:
```

Comment:
Note that the "attributes" in a WebIDL interface are a different sort of thing than DOM attributes. I think it's fine to check in what you have as a base, but we'll want to investigate the right way to say this.

Reply:
Ack